p2p/agent.go

```diff
@@ -223,101 +259,107 @@ func (p *agent) Start(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "error when instantiating Agent host")
 	}
-
-	if err := host.AddBroadcastPubSub(ctx, _broadcastTopic+p.topicSuffix, func(ctx context.Context, data []byte) (err error) {
-		// Blocking handling the broadcast message until the agent is started
-		<-ready
-		var (
-			peerID    string
-			broadcast iotexrpc.BroadcastMsg
-			latency   int64
-		)
-		skip := false
-		defer func() {
-			// Skip accounting if the broadcast message is not handled
-			if skip {
+	networks := []string{_defaultNetwork}
+	if p.networkSeparator != nil {
+		networks = p.networkSeparator.networks
```

Comment:
i think every node need join the default topic, and then join the node specific topic.
topics := []string{_defaultTopic}
if ... {
    topics = append(topics, ...)
}
will better ?

Reply:
also name `_defaultTopic` as `_blockTopic`? which is for broadcasting blocks that everyone will join

Reply:
it's ok